File: client_script.r
# Set up a socket client
con_client <- socketConnection(host = "localhost", port = 9999)
# open(con_client, "r")

# Receive a message from the server
message <- readLines(con_client)
print(message)
while(isIncomplete(con_client)) {
	message <- readLines(con_client)
	print(message)
}
cat("R Client:", message, "\n")

# Clean up
close(con_client)

File: client_script.py
import socket

# Set up a socket client
client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
client_socket.connect(('localhost', 9999))

# Receive a message from the server
message = client_socket.recv(1024).decode()
print("Python Client:", message)

# Clean up
client_socket.close()